src/mpid/ch4/src/ch4_vci.h

```diff
@@ -70,6 +70,142 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_get_vci(int flag, MPIR_Comm * comm_ptr,
 }
 
 #elif MPIDI_CH4_VCI_METHOD == MPICH_VCI__IMPLICIT
+/* Remove the sign bit from an integer, if present */
+MPL_STATIC_INLINE_PREFIX int MPIDI_mask_int_sign(int i)
+{
+    return ((i) & ~(1 << (sizeof(int) * 8 - 1)));
+}
```

Comment:
This is only used with `% MPIDI_global.n_vcis`, right? How about move the `% MPIDI_global.n_vcis` into this function and rename it to `MPIDI_int_to_vci`?

Reply:
Yes. That's a good idea.

Reply:
Done